src/components/Label/__stories__/label.stories.mdx

```diff
@@ -138,5 +140,18 @@ In case of visual overload, use the secondary label in order to create hirarchy.
   </Story>
 </Canvas>
 
+### Clickable
+Labels with an onClick prop change appearance on hover, focus, and click.
+<Canvas>
+  <Story name="Clickable">
+    <div className="monday-storybook-label_block">
+      <div className="monday-storybook-label_inline-block">
+        <Label text="New" kind={Label.kinds.LINE} onClick={NOOP} />
+        <Label text="Beta" onClick={NOOP} />
```

Comment:
This is happening because the filled label is 22px in height and the lined one is 24px in height, they both should be 22px according to spec. I can fix the height and it will resolve this issue as well, or change the clickable element to be the inner element instead and resolve the height in a different PR. Which one sounds better?

Reply:
I fixed the height difference, now the focus works well. Moving the onClick handler to the inner element might not be intuitive IMO, this can also make tests less intuitive. 

But, it will fail Chromatic, though it will now have the correct height

![Screenshot 2023-09-18 at 16 38 38](https://github.com/mondaycom/monday-ui-react-core/assets/18269880/66306b9d-aaeb-461c-aec1-a962ce7f8a53)
![Screenshot 2023-09-18 at 16 38 48](https://github.com/mondaycom/monday-ui-react-core/assets/18269880/c50e9bce-3d70-47a3-8bd5-32d8e14d6369)

Reply:
If this is the behavior Meirav/Yael wants then it's ok it will fail Chromatic, just accept the new baseline on your branch's build